### DISCUSSION TITLE
How can I size-lock flow window?

### DISCUSSION THREAD

**Discussion by Alexnoj** [Flow-Launcher/Flow.Launcher#3067] [Q&A]
Can I hide this mouse sight of resize the flow window?
<img width="489" alt="lock" src="https://github.com/user-attachments/assets/4a791fdd-9dea-4312-87ed-c653e30c65c1">

**Comment by taooceros** [comment]
Don't think so for now. Is there some specific reason for that?

**Reply by Alexnoj** [reply]
Not something special only aesthetic
